h2o-algos/src/main/java/hex/rulefit/Condition.java

```diff
@@ -62,12 +61,18 @@ String constructLanguageCondition() {
 
     @Override
     public boolean equals(Object obj) {
```

Comment:
Can we have a separate test for this?

Reply:
it's tested in RuleFitUtilsTest.java#testConditionAndRule()